packages/vue/src/components/molecules/SfRange/SfRange.vue

```diff
@@ -0,0 +1,231 @@
+    min: {
+      handler(value) {
+        if (this.$refs.range) {
+          return this.updateRangeOptions({
+            range: {
+              min: value >= this.max ? this.max - 1 : value,
+              max: this.max,
+            },
+          });
+        }
+      },
+      immediate: true,
+    },
+    max: {
+      handler(value) {
+        return this.updateRangeOptions({
+          range: {
+            min: this.min,
+            max: value <= this.min ? this.min + 1 : value,
+          },
+        });
+      },
+      immediate: true,
+    },
+    step: {
+      handler(value) {
+        return this.updateRangeOptions({
+          step: value,
+        });
+      },
+      immediate: true,
+    },
+    tooltips: {
+      handler(value) {
+        value
+          ? (value =
+              this.slidersInitialValues.length === 2
+                ? [
+                    wNumb(this.formatTooltipsValues),
+                    wNumb(this.formatTooltipsValues),
+                  ]
+                : [wNumb(this.formatTooltipsValues)])
+          : value;
+        return this.updateRangeOptions({
+          tooltips: value,
+        });
+      },
+      immediate: true,
+    },
+    formatTooltipsValues: {
+      handler(value) {
+        if (this.tooltips)
+          return this.updateRangeOptions({
+            tooltips:
+              this.slidersInitialValues.length === 2
+                ? [
+                    wNumb(this.formatTooltipsValues),
+                    wNumb(this.formatTooltipsValues),
+                  ]
+                : [wNumb(this.formatTooltipsValues)],
+          });
+      },
+      immediate: true,
+    },
+    horizontalOrientation: {
+      handler(value) {
+        value ? (value = "horizontal") : (value = "vertical");
+        return this.resetAndChangeOption({
+          orientation: value,
+        });
+      },
+      immediate: true,
+    },
+    ltrDirection: {
+      handler(value) {
+        value ? (value = "ltr") : (value = "rtl");
+        return this.resetAndChangeOption({ direction: value });
+      },
+      immediate: true,
+    },
+  },
+  mounted() {
+    this.config = {
+      start: this.slidersInitialValues,
+      range: {
+        min: this.min >= this.max ? this.max - 1 : this.min,
+        max: this.max,
+      },
+      step: this.step,
+      connect: true,
+      direction: this.ltrDirection ? "ltr" : "rtl",
+      orientation: this.horizontalOrientation ? "horizontal" : "vertical",
+      behaviour: "tap-drag",
+      tooltips: false,
+      keyboardSupport: true,
+    };
+    this.noUiSliderInit();
+  },
+  methods: {
+    noUiSliderInit(changedValue) {
+      const newConfig = Object.assign(this.config, changedValue);
+      noUiSlider
+        .create(this.$refs.range, newConfig)
+        .on("change", (values) => this.$emit("change", values));
+    },
+    resetAndChangeOption(changedValue) {
+      if (this.$refs.range) {
+        this.$refs.range.noUiSlider.destroy();
```

Comment:
I wonder if we maybe should also destroy it in `beforeDestroy` to prevent memory leak

Reply:
Ok I add it to to beforeDestroy method.